sorbet/rbi/shims/sentry-ruby.rbi

```diff
@@ -0,0 +1,55 @@
+# typed: strong
+# frozen_string_literal: true
+
+module Sentry
```

Comment:
`spoom gem sentry-ruby` generates an empty file, so this is minimal handwritten types for our uses.